joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/LinkingUtil.scala

```diff
@@ -41,11 +36,9 @@ trait LinkingUtil {
   def nodesWithFullName(cpg: Cpg, x: String): mutable.Seq[NodeRef[_ <: NodeDb]] =
     cpg.graph.indexManager.lookup(PropertyNames.FULL_NAME, x).asScala
 
-  /** For all nodes `n` with a label in `srcLabels`, determine the value of `n.\$dstFullNameKey`, use that to lookup the
-    * destination node in `dstNodeMap`, and create an edge of type `edgeType` between `n` and the destination node.
-    */
-  def linkToSingle(
+  def newLinkToSingle(
     cpg: Cpg,
+    srcNodes: List[Node],
     srcLabels: List[String],
```

Comment:
This parameter is no longer used, it can be removed

Reply:
It's getting used at line no 52.